src/javascripts/toggle.js

```diff
@@ -2,93 +2,44 @@
 UK_Parliament.toggle = function() {
 
   var
-    active_state = 'open',
-    toggle_attr = 'data-toggle',
-    toggles = document.querySelectorAll('[' + toggle_attr + ']');
+    attribute = 'data-toggle',
+    buttons = document.querySelectorAll('[' + attribute + '="parent-item"]');
 
   // End program if there are no toggle buttons.
-  if (toggles.length < 1) {
+  if (buttons.length < 1) {
     return false;
   }
 
-  // Add click or change events to toggle buttons.
-  for (var i = 0; i < toggles.length; i++) {
+  // Add click/change events to toggle buttons.
+  for (var i = 0; i < buttons.length; i++) {
 
-    // Only add click/change events to elements with the attribute `data-toggle`
-    // that don't have the value 'item'.
-    if (
-      toggles[i].getAttribute(toggle_attr) !== 'item' &&
-      toggles[i].getAttribute(toggle_attr) !== 'content'
-    ) {
-
-      if (toggles[i].tagName.toUpperCase() === 'INPUT') {
-        switch(toggles[i].getAttribute('type')) {
-        case 'checkbox':
-        case 'radio':
-          toggles[i].onchange = doToggle;
-          break;
-        }
-      } else {
-        toggles[i].onclick = doToggle;
+    if (buttons[i].tagName.toUpperCase() === 'INPUT') {
+      switch(buttons[i].getAttribute('type')) {
+      case 'checkbox':
+      case 'radio':
+        buttons[i].onchange = doToggle;
+        break;
       }
-
+    } else {
+      buttons[i].onclick = doToggle;
     }
 
   }
 
+  // Do toggle
   function doToggle(e) {
     e.preventDefault();
 
     var
-      toggle = e.target,
-      target = undefined,
-      toggle_value = toggle.getAttribute(toggle_attr);
-
-    // Set target element to coresponding DOM-Object based on the clicked toggle
-    // button's value.
-    switch(toggle_value) {
+      active_state = 'open',
+      target = UK_Parliament.traverseUp(e.target, { 'data-toggle': 'item' });
```

Comment:
Should `'data-toggle'` be using the global variable `attribute` that has been declared at line 5

Reply:
I tried doing that but not possible in this case unless we add support for ES6 or if we create an object outside that, add the variable as a property name for the object with its value, then adding the object in that function, like this:

```
var options = {};
options[attribute] = 'item';
var target = UK_Parliament.traverseUp(e.target, options);
```

Which I think is a little bit long-winded. Although this may be a better option, as it's reducing the amount of times we declare a string.

ES6 can easily do it like this:

```
let target = UK_Parliament.traverseUp(e.target, { [attribute]: 'item' });
```